cassandra-persistence/src/test/java/com/netflix/conductor/dao/cassandra/CassandraDAOTest.java

```diff
@@ -69,6 +69,7 @@
     @BeforeClass
     public static void init() throws Exception {
         embeddedCassandra = new EmbeddedCassandra();
+        session = embeddedCassandra.getSession();
     }
 
     @Before
```

Comment:
Please remove `session = embeddedCassandra.getSession();` from the setUp block.

Reply:
Ok; removed.